### DISCUSSION TITLE
possible addon

### DISCUSSION THREAD

**Discussion by jerzean** [GriefPrevention/GriefPrevention#311] [Ideas]
possible addon
is there anyway to change the outline blocks that pop up around a area  
if not that would be a good idea  
be able to change what blocks show up

**Comment by TomLewis** [comment]
Why?

**Comment by bigpresh** [comment]
It would definitely be possible to make it configurable (probably need to be in the core code, not an addon, though) - but I share the same "why?" question, I can't think of any blocks that would be a better choice than the current ones, they stand out well from the rest of the world, and it would potentially lead to confusion for players if GP worked that much differently depending on the server you're on.  I *can* sort of see the point, though, if you have a glowstone floor fetish or something :)

**Comment by RoboMWM** [comment]
Visualization event was added by billy but afaik it's not cancelable - regardless you can do whatever you want in an addon for this.

**Comment by TomLewis** [comment]
The only why I can think of is different coloured visualisations for different players/groups maybe to easily show them in a group.

**Comment by jerzean** [comment]
be able to switch the pumkins/glowstone
example some worlds are mostly sand,    to be able to switch the blocks to something blue/green   would make it stand out in the world that is mostly tan/yellow
